src/content/docs/dev/send-tokens/interchain-tokens/integrate-custom-token.mdx

```diff
@@ -39,42 +39,79 @@ contract MyInterchainToken is ERC20, ERC20Burnable {
 }
 ```
 
-This token is a simple implementation of an ERC-20 token but includes the critical functionality needed to integrate with ITS: the ability to `mint()` and `burn()` tokens. ITS interacts with these functions when bridging tokens between chains.
+This token is a simple implementation of an ERC-20 token that includes the critical functions needed to integrate with ITS: the ability to `mint()` and `burn()` tokens. ITS calls these functions when bridging tokens between chains.
 
-You also have the option to inherit from the [InterchainTokenStandard](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interchain-token/InterchainTokenStandard.sol#L12) so that your token has cross-chain functionality built-in, such as the `interchainTransfer()` function.
+Alternatively, you can inherit from the [InterchainTokenStandard](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interchain-token/InterchainTokenStandard.sol#L12) so that your token has cross-chain functionality built in, such as the `interchainTransfer()` function.
 
-## Integrate the Custom Token with ITS by Deploying a Token Manager
+## Integrate Custom Token with ITS
```

Comment:
Maybe add a brief overview at the top here regarding the steps ie (1. register token, 2. link token) Just as there is a lot going on in this section, could be worth having an overview first.

Reply:
There is an overview there; open the extended collapsable button.